openapi.yaml

```diff
@@ -296,34 +280,20 @@ paths:
       tags:
         - reviews
       parameters:
-        - name: gameId
+        - name: gameIds
           in: query
-          type: string
+          type: array
+          items:
+            type: string
```

Comment:
You should also add the format of this parameter. For example: https://github.com/osu-mist/notes-api/blob/master/openapi.yaml#L65